### PR TITLE
add quantize model link in the doc

### PR DESCRIPTION
**Pull request by dding3**

### CODE REVIEW

File: docs/docs/ProgrammingGuide/image-classification.md [hunk 1]
```diff
@@ -59,11 +59,18 @@ For preprocessors for Image Classification models, please check [Image Classific
...
 * [VGG-19](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_vgg-19_imagenet_0.1.0.model)
+* [VGG-19 Quantize](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_vgg-19-quantize_imagenet_0.1.0.model)
 * [Resnet-50](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_resnet-50_imagenet_0.1.0.model)
+* [Resnet-50 Quantize](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_resnet-50-quantize_imagenet_0.1.0.model)
 * [Densenet-161](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_densenet-161_imagenet_0.1.0.model)
+* [Densenet-161 Quantize](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_densenet-161-quantize_imagenet_0.1.0.model)
 * [Mobilenet](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/imageclassification/imagenet/analytics-zoo_mobilenet_imagenet_0.1.0.model)
```
Comment (jenniew) [inline docs/docs/ProgrammingGuide/image-classification.md]: Is there a Mobilenet quantize model available?

File: docs/docs/ProgrammingGuide/object-detection.md [hunk 2]
```diff
@@ -65,14 +65,20 @@ For preprocessors for Object Detection models, please check [Object Detection Co
...
+* [SSD 300x300 VGG Quantize](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_ssd-vgg16-300x300-quantize_PASCAL_0.1.0.model)
 * [SSD 512x512 VGG](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_ssd-vgg16-512x512_PASCAL_0.1.0.model)
+* [SSD 512x512 VGG Quantize](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_ssd-vgg16-512x512-quantize_PASCAL_0.1.0.model)
 * [Faster-RCNN VGG](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_frcnn-vgg16_PASCAL_0.1.0.model)
 * [Faster-RCNN VGG Compress](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_frcnn-vgg16-compress_PASCAL_0.1.0.model)
+* [Faster-RCNN VGG Compress Quantize](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_frcnn-vgg16-compress-quantize_PASCAL_0.1.0.model)
 * [Faster-RCNN PvaNet](https://s3-ap-southeast-1.amazonaws.com/analytics-zoo-models/object-detection/analytics-zoo_frcnn-pvanet_PASCAL_0.1.0.model)
```
Comment (jenniew) [inline docs/docs/ProgrammingGuide/object-detection.md]: No Faster-RCNN PvaNet quantize model?

File: docs/docs/ProgrammingGuide/image-classification.md (hunk 1 above)
Comment (dding3) [reply docs/docs/ProgrammingGuide/image-classification.md]: Mobilenet model is small and we don't have quantized version.

File: docs/docs/ProgrammingGuide/object-detection.md (hunk 2 above)
Comment (dding3) [reply docs/docs/ProgrammingGuide/object-detection.md]: We have a Faster-RCNN PvaNet compress quantize model

**Comment by jenniew** [comment]
LGTM

**Comment by dding3** [comment]
Thanks for your review. 

Jekins pass. http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/711/

**Pull request merged**